account_asset_management/models/account_asset.py

```diff
@@ -0,0 +1,1092 @@
+        help="Period length for the depreciation accounting entries")
+    method_end = fields.Date(
+        string='Ending Date', readonly=True,
+        states={'draft': [('readonly', False)]})
+    method_progress_factor = fields.Float(
+        string='Degressive Factor', readonly=True,
+        states={'draft': [('readonly', False)]}, default=0.3)
+    method_time = fields.Selection(
+        selection=lambda self: self.env[
+            'account.asset.profile']._selection_method_time(),
+        string='Time Method',
+        required=True, readonly=True,
+        states={'draft': [('readonly', False)]}, default='year',
+        help="Choose the method to use to compute the dates and "
+             "number of depreciation lines.\n"
+             "  * Number of Years: Specify the number of years "
+             "for the depreciation.\n"
+             # "  * Number of Depreciations: Fix the number of "
+             # "depreciation lines and the time between 2 depreciations.\n"
+             # "  * Ending Date: Choose the time between 2 depreciations "
+             # "and the date the depreciations won't go beyond."
+    )
+    prorata = fields.Boolean(
+        string='Prorata Temporis', readonly=True,
+        states={'draft': [('readonly', False)]},
+        help="Indicates that the first depreciation entry for this asset "
+             "have to be done from the depreciation start date instead "
+             "of the first day of the fiscal year.")
+    depreciation_line_ids = fields.One2many(
+        comodel_name='account.asset.line',
+        inverse_name='asset_id',
+        string='Depreciation Lines', copy=False,
+        readonly=True, states={'draft': [('readonly', False)]})
+    type = fields.Selection(
+        selection=[('view', 'View'),
+                   ('normal', 'Normal')],
+        string='Type',
+        required=True, readonly=True, default='normal',
+        states={'draft': [('readonly', False)]})
+    company_id = fields.Many2one(
+        comodel_name='res.company',
+        string='Company', required=True, readonly=True,
+        default=lambda self: self._default_company_id())
+    company_currency_id = fields.Many2one(
+        comodel_name='res.currency',
+        related='company_id.currency_id',
+        string='Company Currency',
+        store=True, readonly=True)
+    account_analytic_id = fields.Many2one(
+        comodel_name='account.analytic.account',
+        string='Analytic account',
+        domain=[('type', '!=', 'view'),
+                ('state', 'not in', ('close', 'cancelled'))])
+
+    @api.model
+    def _default_company_id(self):
+        return self.env['res.company']._company_default_get('account.asset')
+
+    @api.multi
+    def _compute_move_line_check(self):
+        for asset in self:
+            for line in asset.depreciation_line_ids:
+                if line.move_id:
+                    asset.move_line_check = True
+                    continue
```

Comment:
seems like `continue` is not needed here

Reply:
Very good catch :+1: